2023-04-19 06:28:45,027 WARNING Not Found: /news2 /home/i/.local/lib/python3.10/site-packages/django/utils/log.py None
2023-04-19 06:28:45,028 WARNING "GET /news2 HTTP/1.1" 404 2524 /home/i/.local/lib/python3.10/site-packages/django/core/servers/basehttp.py None
2023-04-19 06:29:14,752 INFO "GET /news/ HTTP/1.1" 200 8320 /home/i/.local/lib/python3.10/site-packages/django/core/servers/basehttp.py None
2023-04-19 06:29:19,414 WARNING Not Found: /news2 /home/i/.local/lib/python3.10/site-packages/django/utils/log.py None
2023-04-19 06:29:19,415 WARNING "GET /news2 HTTP/1.1" 404 2524 /home/i/.local/lib/python3.10/site-packages/django/core/servers/basehttp.py None
2023-04-19 06:31:44,619 WARNING Not Found: /news2 /home/i/.local/lib/python3.10/site-packages/django/utils/log.py None
2023-04-19 06:31:44,620 WARNING "GET /news2 HTTP/1.1" 404 2524 /home/i/.local/lib/python3.10/site-packages/django/core/servers/basehttp.py None
2023-04-19 06:46:15,594 WARNING Not Found: /news2/ /home/i/.local/lib/python3.10/site-packages/django/utils/log.py None
2023-04-19 06:46:15,595 WARNING "GET /news2/ HTTP/1.1" 404 2527 /home/i/.local/lib/python3.10/site-packages/django/core/servers/basehttp.py None
2023-04-19 06:47:01,006 WARNING Not Found: /news2/ /home/i/.local/lib/python3.10/site-packages/django/utils/log.py None
2023-04-19 06:47:01,007 WARNING "GET /news2/ HTTP/1.1" 404 2527 /home/i/.local/lib/python3.10/site-packages/django/core/servers/basehttp.py None
